com.unity.netcode.gameobjects/Editor/NetworkAnimatorEditor.cs

```diff
@@ -1,113 +1,23 @@
-using System;
 using Unity.Netcode.Components;
 using UnityEditor;
-using UnityEditor.Animations;
 using UnityEngine;
 
 namespace Unity.Netcode.Editor
 {
-    public static class TextUtility
-    {
-        public static GUIContent TextContent(string name, string tooltip)
-        {
-            var newContent = new GUIContent(name);
-            newContent.tooltip = tooltip;
-            return newContent;
-        }
-
-        public static GUIContent TextContent(string name)
-        {
-            return new GUIContent(name);
-        }
-    }
-
     [CustomEditor(typeof(NetworkAnimator), true)]
     [CanEditMultipleObjects]
     public class NetworkAnimatorEditor : UnityEditor.Editor
     {
-        private NetworkAnimator m_AnimSync;
-        [NonSerialized] private bool m_Initialized;
-        private SerializedProperty m_AnimatorProperty;
-        private GUIContent m_AnimatorLabel;
-
-        private void Init()
-        {
-            if (m_Initialized)
-            {
-                return;
-            }
-
-            m_Initialized = true;
-            m_AnimSync = target as NetworkAnimator;
-
-            m_AnimatorProperty = serializedObject.FindProperty("m_Animator");
-            m_AnimatorLabel = TextUtility.TextContent("Animator", "The Animator component to synchronize.");
-        }
-
         public override void OnInspectorGUI()
         {
-            Init();
```

Comment:
Doesn't seem worth it to me to go to the trouble of tracking init and not re-drawing.  This isn't remotely performance noticeable, and makes the code harder to grok.